android/src/main/java/com/tailscale/ipn/StartVPNWorker.java

```diff
@@ -45,7 +45,7 @@ public Result doWork() {
         String channelId = "start_vpn_channel";
 
         // Use createNotificationChannel method from App.java
-        app.createNotificationChannel(channelId, "Start VPN Channel", NotificationManager.IMPORTANCE_DEFAULT);
+        app.createNotificationChannel(channelId, getApplicationContext().getString(R.string.vpn_start), getApplicationContext().getString(R.string.notifications_delivered_when_user_interaction_is_required_to_establish_the_vpn_tunnel), NotificationManager.IMPORTANCE_MAX);
```

Comment:
https://developer.android.com/reference/android/app/NotificationManager#IMPORTANCE_MAX - this says it's unused?

Reply:
Yes, but we do want the behaviour of `IMPORTANCE_HIGH`: "Higher notification importance: shows everywhere, makes noise and peeks. May use full screen intents.". 

`IMPORTANCE_MAX` will get us that, in addition to whatever they will ever come up with next to make these notifications more prominent.

Reply:
Didn't notice it was _deprecated_. Fixed.

Reply:
https://android.googlesource.com/platform/frameworks/base.git/+/master/core/java/android/app/NotificationManager.java#454 IMPORTANCE_MAX not being used right now